x/farming/types/params.go

```diff
@@ -71,7 +87,55 @@ func validatePrivatePlanCreationFee(i interface{}) error {
 	}
 
 	if v.Empty() {
-		return fmt.Errorf("plan creation fee must not be empty")
+		return fmt.Errorf("private plan creation fee must not be empty")
```

Comment:
By allowing zero coins in the validation process, we can also consider configuring environments that do not receive fees ( `PrivatePlanCreationFee`, and also `StakingCreationFee` ) but, Let's leave it like this for now and test it with zero coins later and fix it.